backend/btrixcloud/crawlmanager.py

```diff
@@ -393,3 +396,37 @@ async def update_scheduled_job(
         await self.create_from_yaml(data, self.namespace)
 
         return cron_job_id
+
+    async def create_replica_deletion_scheduled_job(
+        self,
+        job_id: str,
+        params: Dict[str, object],
+        delay_days: int,
+    ) -> Tuple[str, Optional[str]]:
+        """create scheduled job to delay replica file in x days"""
+        now = dt_now()
+        run_at = now + timedelta(days=delay_days)
+        schedule = f"0 0 {run_at.day} {run_at.month} *"
```

Comment:
```suggestion
        schedule = f"{run_at.minute} {run_at.hour} {run_at.day} {run_at.month} *"
```
I think this should be a full day from moment of deletion, to avoid scenarios where its still deleted quickly if its at the end of the day.